docs/src/operate/registration/index.md

```diff
@@ -1,11 +1,21 @@
 ---
-title: Registration
+title: Entity Management
```

Comment:
Open to use any other title like `Entity Lifecycle Management` that covers CRUD operations on entities. Even the side-pane title (directory name) which is currently `Registration` could be changed accordingly.